default/python/AI/MilitaryAI.py

```diff
@@ -46,6 +46,27 @@ def cur_best_mil_ship_rating(include_designs=False):
     return max(best_rating, 0.001)
 
 
+def get_preferred_max_military_portion_for_single_battle():
+    """
+    Determine and return the preferred max portion of military to be allocated to a single battle.  May be used to
```

Comment:
Multi-line docstrings should have a single-line summary followed by a blank line (cf. [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings))